src/main/webapp/view/discussion/discussion-comment.js

```diff
@@ -106,27 +108,35 @@ export default class DiscussionComment extends HTMLElement {
 
   /**
    * Posts the content of the reply textarea as a reply to this comment,
-   * and reloads the discussion area.
+   * and resets the reply form.
    */
   postReplyClicked() {
     const textarea = this.shadowRoot.querySelector(
```

Comment:
Maybe this should be renamed to reply text area, could lead to confusion between which text area you are referring to

Reply:
i'll leave it for another time...